src/test/scala/parsec/HListParsersSpec.scala

```diff
@@ -0,0 +1,136 @@
+package parsec
+
+import scala.language.postfixOps
+import util.ParserFunSuite
+
+/** Spec for `HListParsef`
+ *
+ *  The idea is that each parser lives as a separate tuple */
+class HListParserSpec
```

Comment:
Would you also suggest renaming the other test files to `Spec` rather than `Suite`? I don't mind what the name is as long as we are consistent :-). Since we extend `FunSuite` from `scalatest` should we keep the `Suite` suffix?